Practice1/src/index.html

```diff
@@ -0,0 +1,218 @@
+                <button class="example-button" type="submit" class="search-btn">
+                    <i class="fa-solid fa-magnifying-glass"></i></button>
+            </form>
+        </div>
+        <div class="header-container">
+            <div class="content">
+                <img class="header-container-img" src="./assets/images/hospital-img.jpg" alt="Hospitals">
+                <p class="content-p">Find Clinics and Hospitals</p>
+            </div>
+            <div class="content">
+                <img class="header-container-img" src="./assets/images/pharmacy-img.jpg" alt="Hospitals">
+                <p class="content-p">Find Pharmacies</p>
+            </div>
+            <div class="content">
+                <img class="header-container-img" src="./assets/images/article-img.jpg" alt="Hospitals">
+                <p class="content-p">Read Heath Articles</p>
+            </div>
+        </div>
+    </header>
+    <section class="page-how-it-work">
+        <h3 class="heading">How It Works</h3>
+        <div class="blog-container">
+            <div class="blog-container-content">
+                <img class="blog-container-img" src="./assets/images/search-img.jpg" alt="Search">
+                <p class="font">Search</p>
+                <p class="font-little">Search for a doctor or health specialist or enter a particular condition
+                    or procedure. We cover 47 medical specialities
+                </p>
+            </div>
+            <div class="blog-container-content">
+                <img class="blog-container-img" src="./assets/images/compare-img.jpg" alt="Compare">
+                <p class="font">Compare</p>
+                <p class="font-little" class="font-compare">Compate medical centers using verified patient reviews and
+                    filter for
+                    distance, availability and insure to find the right doctor for you
+                </p>
+            </div>
+            <div class="blog-container-content">
+                <img class="blog-container-img" src="./assets/images/call-img.jpg" alt="Call">
+                <p class="font">Call</p>
+                <p class="font-little">After having found the right doctor and the insurance company, you can
+                    placea phone call to the venue and manager the booking.
+                </p>
+            </div>
+        </div>
+    </section>
+    <section class="page-read-health">
+        <img class="page-read-health-img" src="./assets/images/page-read-health-img.png" alt="background">
+        <h3 class="page-read-health-heading">Read Health Articles</h3>
+        <div class="page-read-health-container">
+            <div class="mask">
+                <div>
+                    <img class="mask-img" src="./assets/images/mask-img.png" alt="">
+                </div>
+                <div class="page-read-health-blog">
+                    <p class="top-medical-input">Top medical Proffessionals</p>
```

Comment:
Should it be a heading instead of paragraph?

Reply:
Fixed